packages/picker/src/Picker.ts

```diff
@@ -412,7 +432,7 @@ export class PickerBase extends SizedMixin(Focusable) {
 
     protected render(): TemplateResult {
         return html`
-            ${this.renderButton} ${this.renderPopover}
+            ${this.renderButton}
```

Comment:
Would it make sense to get rid of this function and rename `renderButton` to `render`?